app/assets/stylesheets/layout/common/_header.scss

```diff
@@ -1,7 +1,7 @@
 .l-header {
-  background: $color-green-primary;
-  min-height: $baseline-unit*11;
-  position: relative;
+  width: 100%;
+  z-index: 200;
+  min-height: 55px;
```

Comment:
I agree that would be better but the problem is that we are mixing units (px and rem) which we could do with calc() in CSS which is not generally used on the site. There's also the px() SASS function which converts px to em which may be exploitable somehow.